changelogs/unreleased/gh-9556-fix-uri-ipv6.md

```diff
@@ -1,4 +1,5 @@
 ## bugfix/uri
```

Comment:
> NO_DOC=no doc feature

What does it mean?

Anyway, I would just mark it as a bugfix: `NO_DOC=bugfix`.